src/app/menu/menu.component.ts

```diff
@@ -1,28 +1,37 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
 import { WanikaniTokenService, UserData, UserService } from 'wanikani-api-ng';
 import { Observable, Subject } from 'rxjs';
 import { map, filter, takeUntil, switchMap, tap } from 'rxjs/operators';
-import { PopoverController } from '@ionic/angular';
+import { PopoverController, ModalController } from '@ionic/angular';
 import { UserInfoComponent } from '../user-info/user-info.component';
+import { SettingsPage } from '../settings/settings.page';
 
 @Component({
   selector: 'app-menu',
   templateUrl: './menu.component.html',
   styleUrls: ['./menu.component.scss'],
 })
-export class MenuComponent implements OnInit {
+export class MenuComponent implements OnInit, OnDestroy {
 
   user: Observable<UserData>;
   userLoaded = new Subject();
-  constructor(private tokenService: WanikaniTokenService, private userService: UserService, private popoverController: PopoverController) { }
+  constructor(private tokenService: WanikaniTokenService, 
+              private userService: UserService, 
+              private popoverController: PopoverController,
+              private modalController: ModalController) { }
 
   ngOnInit() {
+    console.log('menu init')
```

Comment:
These log messages should be removed

Reply:
removed, also remove the OnDestroy stuff as it wasn't doing anything